### PR TITLE
Moar tests

### PR DESCRIPTION
**Pull request by stkent**

### CODE REVIEW

File: amplify/src/main/java/com/github/stkent/amplify/tracking/ClockUtil.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.github.stkent.amplify.tracking;
+
+import android.support.annotation.Nullable;
+
+public final class ClockUtil {
+
+    @Nullable
+    private static Long fakeCurrentTimeMillis = null;
```
Comment (btkelly) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/ClockUtil.java]: This seems strange as it's a production class

**Comment by btkelly** [comment]
Also license check, I'll work on auto adding licenses next

File: amplify/src/main/java/com/github/stkent/amplify/tracking/ClockUtil.java (hunk 1 above)
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/ClockUtil.java]: Yeah, this is why it's a hack vs. proper injection.

Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/tracking/ClockUtil.java]: Can fix if you think it's filthy

**Comment by stkent** [comment]
Can the license check report when failing the build? Is that doable with gnag?

**Pull request merged**